src/warnet/lnnode.py

```diff
@@ -78,17 +81,21 @@ def generate_cli_command(self, command: list[str]):
         return cmd
 
     def export(self, config, subdir):
-        container_name = self.backend.get_container_name(self.tank.index, ServiceType.LIGHTNING)
+        container_name = self.backend_interface.get_container_name(
```

Comment:
i think this already cached as `self.hostname` ?

Reply:
Switched to using `self.hostname` here and later in the function.